Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    killbill-cybersource (5.0.1)
+    killbill-cybersource (5.0.2)
```

Comment:
No need to change the version yet. The version bump is done by the build environment, right before a release (tag).